types/linkifyjs/react.d.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { LinkifyOptions } from "./index";
 
-declare interface LinkifyProps {
+interface LinkifyProps {
```

Comment:
oh thanks

Reply:
Thank ESLint for that 😄 Apparently declare is useless in this context.